src/code42cli/cmds/devices.py

```diff
@@ -154,11 +155,11 @@ def _change_device_name(sdk, guid, name):
 @devices.command()
 @device_guid_argument
 @sdk_options()
-def show(state, device_guid, format=None):
+def show(state, device_guid):
     """Print individual device details. Requires device GUID."""
 
-    formatter = OutputFormatter(format, _device_info_keys_map())
-    backup_set_formatter = OutputFormatter(format, _backup_set_keys_map())
+    formatter = OutputFormatter(OutputFormat.TABLE, _device_info_keys_map())
```

Comment:
why is `TABLE` the only valid output format for this command?

Reply:
show commands only output a single record and we don't have any in the CLI that allow other formats for show commands.

For some reason, there was a format param here but there was no `format_option` so it just always uses `None` (`TABLE`) anyway